homeassistant/components/template/light.py

```diff
@@ -389,6 +429,24 @@ def update_brightness(self):
             _LOGGER.error(ex)
             self._state = None
 
+    @callback
+    def update_white_value(self):
+        """Update the white value from the template."""
+        if self._white_value_template is None:
+            return
+        try:
+            white_value = self._white_value_template.async_render()
+            if 0 <= int(white_value) <= 255:
```

Comment:
We need to add a check for `"None"` and empty string here.
```suggestion
            if white_value in ("None", ""):
                self._white_value = None
                return
            if 0 <= int(white_value) <= 255:
```